Demo/EarlGreyExample/EarlGreyExampleSwiftTests/EarlGreyExampleSwiftTests.swift

```diff
@@ -195,9 +195,8 @@ class EarlGreyExampleSwiftTests: XCTestCase {
 
   func testWithCondition() {
     let myCondition = GREYCondition.init(name: "Example condition", block: { () -> Bool in
-      var i = 1
-      while i <= 100000 {
-        i++
+      for j in 0...100000 {
```

Comment:
'++' is deprecated in Swift 2.2 and is [to be removed](https://github.com/apple/swift-evolution/blob/master/proposals/0004-remove-pre-post-inc-decrement.md) in Swift 3.